### PR TITLE
Fixes to support MySQL 8 due to specific reserved keywords

### PR DESCRIPTION
**Pull request by galovics**
## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [x] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [x] Create/update unit or integration tests for verifying the changes made.

- [x] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [x] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [x] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: .github/workflows/build-mysql.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        run: |
+            ./gradlew --no-daemon -q createMySQLDB -PdbName=fineract_tenants
+            ./gradlew --no-daemon -q createMySQLDB -PdbName=fineract_default
+      - name: Install additional software
+        run: |
+            sudo apt-get update
+            sudo apt-get install ghostscript -y
+
+      - name: Basic Auth Build & Test
+        run: ./gradlew --no-daemon -q --console=plain licenseMain licenseTest check build test --fail-fast doc -x :twofactor-tests:test -x :oauth2-test:test -PdbType=mysql
```
Comment (vidakovic) [inline .github/workflows/build-mysql.yml]: When you rebase after #2313 is merged into develop then this line should look like:

```
./gradlew --no-daemon -q --console=plain build test --fail-fast doc -x :twofactor-tests:test -x :oauth2-test:test -PdbType=mysql
```

**Review by vidakovic — APPROVED** [review]
LGTM

**Comment by vidakovic** [comment]
@galovics I think the easiest approach would be to merge first #2313, because that solves very likely those 2 build errors.

File: .github/workflows/build-mysql.yml (hunk 1 above)
Comment (galovics) [reply .github/workflows/build-mysql.yml]: Done.

**Review by vidakovic — APPROVED** [review]
LGTM

**Pull request merged**